aten/src/ATen/cudnn/cudnn-wrapper.h

```diff
@@ -5,9 +5,10 @@
 #define STRINGIFY(x) #x
 #define STRING(x) STRINGIFY(x)
 
-#if CUDNN_MAJOR < 6
-#pragma message ("CuDNN v" STRING(CUDNN_MAJOR) " found, but need at least CuDNN v6. You can get the latest version of CuDNN from https://developer.nvidia.com/cudnn or disable CuDNN with USE_CUDNN=0")
-#pragma message "We strongly encourage you to move to 6.0 and above."
+#if CUDNN_MAJOR < 8
+#pragma message("CuDNN v" STRING( \
+    CUDNN_MAJOR) " found, but need at least CuDNN v8. You can get the latest version of CuDNN from https://developer.nvidia.com/cudnn or disable CuDNN with USE_CUDNN=0")
+#pragma message "We strongly encourage you to move to 8.0 and above."
```

Comment:
Tiny nit, might want to match the README's specification of 8.5 here but otherwise LGTM https://github.com/pytorch/pytorch?tab=readme-ov-file#nvidia-cuda-support